flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/api/input/StreamOperatorContextBuilder.java

```diff
@@ -108,11 +108,15 @@ StreamOperatorStateContext build(Logger logger) throws IOException {
 
         StateBackend stateBackend;
         try {
+            Configuration jobConfig = environment.getJobConfiguration();
+            jobConfig.set(StateChangelogOptions.ENABLE_STATE_CHANGE_LOG, false);
+            Configuration clusterConfig = new Configuration(configuration);
+            clusterConfig.set(StateChangelogOptions.ENABLE_STATE_CHANGE_LOG, false);
```

Comment:
IIUC, the cluster config is possible to be true currently. But we always set it to false here.